npm2deb/__init__.py

```diff
@@ -576,3 +577,21 @@ def _get_github_url_from_git(self, url):
         if result == 'undefined':
             result = None
         return result
+
+    def create_git_repository(self):
+        if _os.system('gbp import-dsc --help 1>/dev/null') == 0:
+            dsc_name = None
+            _os.chdir('..')
```

Comment:
Doesn't it need to undo this `cd` at the end ?

Reply:
Not really, since the directory it points to before `cd ..` is `module-name`-`upstream-version` which is being removed if `gbp import-dsc` is done successfully.